src/tests/EdFi.DataManagementService.Tests.E2E/Features/SuperclassReferenceValidation.feature.disabled

```diff
@@ -0,0 +1,116 @@
+                      }
+                  }
+                  """
+             Then it should respond with 201
+
+        @ignore
+        Scenario: Ensure clients cannot create a Program that references a non-existing Education Organization
+            Given educationOrganizationId 102
+             When a POST request is made to "ed-fi/programs" with
+                  """
+                  {
+                      "programName": "Program Non-Existing Test",
+                      "programTypeDescriptor": "uri://ed-fi.org/ProgramTypeDescriptor#Bilingual",
+                      "educationOrganizationReference": {
+                          "educationOrganizationId": 102
+                      }
+                  }
+                  """
+             Then it should respond with 403
+              And the response body is
+                  """
+                  {
+                      "detail": "Access to the requested data could not be authorized.",
+                      "type": "urn:ed-fi:api:security:authorization",
+                      "title": "Authorization Denied",
+                      "status": 403,
+                      "correlationId": null,
+                      "errors": [
+                          "No relationships have been established between the caller's education organization id claims (255901, 19255901) and the resource item's 'EducationOrganizationId' value."
+                      ]
+                  }
+                  """
+        @ignore
+        Scenario: Ensure clients can update a program that references to an existing school so it references now the Local Education Agency
+            Given programName Program School Test
+            #set value to {id}
+             When a PUT request is made to "ed-fi/programs/{id}" with
+                  """
+                  {
+                      "programName": "Program School Test",
+                      "programTypeDescriptor": "uri://ed-fi.org/ProgramTypeDescriptor#Bilingual",
+                      "educationOrganizationReference": {
+                          "educationOrganizationId": 101
+                      }
+                  }
+                  """
+             Then it should respond with 204
+
+        @ignore
+        Scenario: Ensure clients cannot update a program that references to an existing Local Agency so it references now a Non Existing Education Organization
+            Given programName Program LEA Test
+             #set value to {id}
+             When a PUT request is made to "ed-fi/programs/{id}" with
+                  """
+                  {
+                      "programName": "Program LEA Test",
+                      "programTypeDescriptor": "uri://ed-fi.org/ProgramTypeDescriptor#Bilingual",
+                      "educationOrganizationReference": {
+                          "educationOrganizationId": 102
+                      }
+                  }
+                  """
+             Then it should respond with 40
```

Comment:
Should be 409, with the same message that you'll have on the POST request.

Reply:
Changed